app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
   MetaMetricsController: {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    eventsBeforeMetricsOptIn: true,
   },
   NetworkController: {
-    networkId: true,
     networkStatus: true,
+    selectedNetworkClientId: false,
+    networkId: true,
     providerConfig: {
-      nickname: true,
-      ticker: true,
+      rpcUrl: false,
       type: true,
+      chainId: false,
+      ticker: true,
+      nickname: true,
+      rpcPrefs: false,
+      id: false,
```

Comment:
This is a UUID that might be referenced elsewhere in state eventually, so it seems safe to include. It wouldn't be useful today, but it might be later.

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b3330828235d61e68ab356e3e859a30d3c44af6f